modules/installation-configuration-parameters.adoc

```diff
@@ -1278,9 +1283,77 @@ Optional VMware vSphere machine pool configuration parameters are described in t
 
 endif::vsphere,vmc[]
 
+ifdef::alibaba[]
+//From: https://github.com/openshift/installer/blob/master/data/data/install.openshift.io_installconfigs.yaml#L20; https://github.com/openshift/openshift-docs/pull/40651/files#r792388476
+
+[id="installation-configuration-parameters-additional-alibaba_{context}"]
+== Additional {alibaba} configuration parameters
+
+Additional {alibaba} configuration parameters are described in the following table:
+
+.Optional {alibaba} parameters
+[cols=".^2m,.^3a,^5a",options="header"]
+|====
+|Parameter|Description|Values
+
+|`platform.alibabacloud.defaultMachinePlatform.description`
```

Comment:
I don't see this field here https://github.com/openshift/installer/blob/4fc9fa88c22221b6cede2456b1c33847943b75c9/pkg/types/alibabacloud/machinepool.go#L19

Reply:
@kwoodson I got that from https://github.com/openshift/installer/blob/master/data/data/install.openshift.io_installconfigs.yaml#L1131

Reply:
@mburke5678 I guess https://github.com/openshift/installer/blob/master/data/data/install.openshift.io_installconfigs.yaml#L108-L136 instead.

Reply:
@jianli-wei @kwoodson I updated the list to match the parameters in the `install.openshift.io_installconfigs.yaml` as Jianli indicated.